Jenkinsfile

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent {
+        label 'slave-group-k8s'
+    }
+
+    environment {
+        GO111MODULE = 'on'
+        KIND_VERSION = 'v0.11.0'
+        METALLB_VERSION = 'v0.9.6'
+        KUBECONFIG = 'kind-kube-config.yaml'
```

Comment:
@ryanemerson seem to $KUBECONFIG should be path + file, not just file

Reply:
Thanks @dmvolod, that seems to have been the issue :+1: